yotiprofile.go

```diff
@@ -2,20 +2,21 @@ package yoti
 
 import (
 	"github.com/getyoti/yoti-go-sdk/attribute"
+	"github.com/lampkicking/core-api-go/attributenames"
 )
 
 const (
-	attrConstSelfie                  = "selfie"
-	attrConstGivenNames              = "given_names"
-	attrConstFamilyName              = "family_name"
-	attrConstFullName                = "full_name"
-	attrConstMobileNumber            = "phone_number"
-	attrConstEmailAddress            = "email_address"
-	attrConstDateOfBirth             = "date_of_birth"
-	attrConstAddress                 = "postal_address"
-	attrConstStructuredPostalAddress = "structured_postal_address"
-	attrConstGender                  = "gender"
-	attrConstNationality             = "nationality"
+	attrConstSelfie                  = attributenames.Selfie
```

Comment:
Why rem-map them? Why not just use them as provided from the attributenames pkg?

Reply:
It was originally because they were referenced in the tests too, but I will just change them there too to use them straight from the attributenames pkg

Reply:
Yeah, should be able to rename them in place to the attributenames.<> and then once done should be able to delete this. Should result in more maintainable code in the long run. (one source of truth)